.gitignore

```diff
@@ -5,3 +5,4 @@ Cargo.lock
 .DS_Store
 
 .vscode
+act.sh
```

Comment:
Omit this if this was specific for your usecase

```suggestion
```

Reply:
Sorry about this, I was using the file for running actions locally.